java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -572,12 +585,18 @@ private static Object[] bitmapCommands(BaseTransaction<?> transaction) {
                     1L, // bitpos(key, 1)
                     25L, // bitpos(key, 1, 3)
                     25L, // bitpos(key, 1, 3, 5)
+                    new Long[] {
+                        6L
+                    }, // bitfieldReadOnly(key1, new BitFieldReadOnlySubCommands[]{new BitFieldGet(new
+                    // SignedEncoding(5), new Offset(3))})
```

Comment:
You can keep the comment simple here, so it will fit one line
```suggestion
                    new Long[] { 6L }, // bitfieldReadOnly(key1, [BitFieldGet(SignedEncoding(5), Offset(3))])
```